monitor/README.md

```diff
@@ -0,0 +1,15 @@
+# Cilium node monitor
+
+The node monitor provides an API for reading the events from the BPF datapath.
+When the process `cilium-node-monitor` is started it handles new connections to
+`$RuntimePath/monitor.sock`. Users of the API are expected to read the
+[Meta][0] and [Payload][1]. Since the payload can vary in size to make decoding
+easier the Meta contains the size of the payload.
+
+Notifications from the BPF datapath are transmitted via the perf ring buffer.
+The perf ring buffer is a single reader data structure. The node monitor
+provides access to the notifications to multiple readers by multiplexing all
+notifications to all registered readers.
+
```

Comment:
A short paragraph explaining that the monitor is built as part of the main cilium build and that it is installed with it (I think this is always true? I'm not sure about the container images we provide, but it is probably the case). If the user wishes to build it separately, they can invoke make on this Makefile (I think this will work, the file includes the top level defs file and I think that's all it needs.